init.c

```diff
@@ -257,7 +257,9 @@ static void print_help()
 	  "                                additional -a adds backup trackers\n"
 	  "-c, --comment=<comment>       : add a comment to the metainfo\n"
 	  "-d, --no-date                 : don't write the creation date\n"
-	  "-f, --force                   : overwrite output file if it exists\n"
+	  "-e, --exclude=<pat>[,<p>]*    : exclude files whose name matches the pattern <p>\n"
```

Comment:
Please don't mix the metavars and use either `<p>` or `<pat>`, not both. Excuse me, if my earlier comment was misleading.

Reply:
Went with <pat> as it's a bit more descriptive